themes/navy/source/css/_partial/index.styl

```diff
@@ -45,6 +45,39 @@
   @media mq-normal
     display: block
 
+#intro-news-list
+  background-color: #10171e
+  margin: -1px 0
+
+#intro-news-flex
+  display: flex
+  flex-flow: column
+  @media mq-normal
+    flex-flow: row wrap
+
+.intro-news-wrap
+  padding: 36px 30px
```

Comment:
I think `36px` is a little too high for a smartphone. How do you think?

![navy](https://user-images.githubusercontent.com/11273093/66391650-a1668800-ea08-11e9-889d-f0d5da7d1428.jpg)

Reply:
Now it is `24px` for smartphone, and remains `36px` on tablet & desktop.